senator/game.py

```diff
@@ -84,16 +105,45 @@ def nash_conv(self) -> float:
 
 
 class Game:
-    def __init__(self, initial_utility: np.ndarray, votes: np.ndarray):
-        assert len(initial_utility.shape) == 2, "Initial utility should be a 2D array"
+    def __init__(
+        self,
+        initial_utility: np.ndarray,
+        votes: np.ndarray,
+        is_owner: np.ndarray,
+        num_iterations: int = 7,
```

Comment:
Souhlasim s Otrokem, že `iterations` už je použité pro to řešení pomocí regret matchingu (i když ta fce se jmenuje `rm_step`...), takže to chce nějak sjednotit a použít někde step a jinde konzistentně iteration.

Reply:
Ok, dal jsem num_step